components/latex_block/latex_block.tsx

```diff
@@ -27,7 +27,7 @@ export default class LatexBlock extends React.PureComponent<Props, State> {
 
     componentDidMount(): void {
         import('katex').then((katex) => {
-            this.setState({katex});
+            this.setState({katex: katex.default});
```

Comment:
pushed a small commit with this change in both inline and block. Due to upgrade 0.14.0 ( https://github.com/KaTeX/KaTeX/releases?page=2 ), the returned value from the dynamic import is a module compared to the past. cc: @hmhealey @crspeller for a 👍  or 👎  - I debugged the exception that led to latex not compiling and manually tested and everything works properly.